tools/lib/url_file.py

```diff
@@ -57,6 +57,8 @@ def __exit__(self, exc_type, exc_value, traceback):
   def get_length_online(self):
     timeout = Timeout(connect=50.0, read=500.0)
     response = self._http_client.request('HEAD', self._url, timeout=timeout, preload_content=False)
+    if not (response.status >= 200 and response.status <= 299):
```

Comment:
```suggestion
    if not (200 <= status <= 299):
```

This seems easier to read